src/app/components/Avatar/Avatar.module.css

```
@@ -0,0 +1,6 @@
+.img {
```

I would prefer to call this class like the component:

```suggestion
.Avatar {
```

or 

```suggestion
.avatar {
```